pkg/flowaggregator/flowaggregator.go

```diff
@@ -294,6 +299,27 @@ func (fa *flowAggregator) InitAggregationProcess() error {
 	return err
 }
 
+func (fa *flowAggregator) createAndSendTemplate(isRecordIPv6, isOriginExporterIPv6 bool) (uint16, error) {
+	templateID := fa.exportingProcess.NewTemplateID()
+	recordIPFamily := "IPv4"
+	exporterIPFamily := "IPv4"
+	if isRecordIPv6 {
+		recordIPFamily = "IPv6"
+	}
+	if isOriginExporterIPv6 {
+		exporterIPFamily = "IPv6"
```

Comment:
what the origin exporter, which IP is that?

which IEs are impacted by exporterIPFamily == IPv6 vs exporterIPFamily == IPv4?

Reply:
There is a field called originalExporterIPv4Address/originalExporterIPv6Address in flow record, which is the IP address of exporter (in our case node ip). IP family of original exporter may not be the same as that of source/destination IP in the same flow record. So we need to send different templates to cover these scenarios.